diff --git a/node_modules/@custom-elements-manifest/analyzer/src/features/post-processing/apply-inheritance.js b/node_modules/@custom-elements-manifest/analyzer/src/features/post-processing/apply-inheritance.js
index bd4e9b1..804c483 100644
--- a/node_modules/@custom-elements-manifest/analyzer/src/features/post-processing/apply-inheritance.js
+++ b/node_modules/@custom-elements-manifest/analyzer/src/features/post-processing/apply-inheritance.js
@@ -52,11 +52,7 @@ export function applyInheritancePlugin() {
                 customElement[type] = customElement?.[type]?.map(item => item.name === existing.name
                   ? {
                       ...newItem,
-                      ...existing,
-                      ...{
-                        ...(newItem.type ? { type: newItem.type } : {}),
-                        ...(newItem.privacy ? { privacy: newItem.privacy } : {})
-                      }
+                      ...existing
                     }
                   : item);
               } else {
