diff --git a/node_modules/vite-plugin-checker/dist/cjs/checkers/typescript/main.js b/node_modules/vite-plugin-checker/dist/cjs/checkers/typescript/main.js
index d00bb7b..79730e6 100644
--- a/node_modules/vite-plugin-checker/dist/cjs/checkers/typescript/main.js
+++ b/node_modules/vite-plugin-checker/dist/cjs/checkers/typescript/main.js
@@ -114,7 +115,7 @@ const createDiagnostic = (pluginConfig) => {
           void 0,
           reportWatchStatusChanged
         );
-        import_typescript.default.createSolutionBuilderWithWatch(host, [configFile], {}).build();
+        import_typescript.default.createSolutionBuilderWithWatch(host, [configFile], {emitDeclarationOnly: true}).build();
       } else {
         const host = import_typescript.default.createWatchCompilerHost(
           configFile,
